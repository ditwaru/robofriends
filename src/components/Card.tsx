import React from 'react';
import { RobotProps } from '../types/RobotTypes';

const Card: React.FunctionComponent<RobotProps> = ({ name, id, email }) => {
  return (
    <div className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5 tc">
      <img
        src={`https://robohash.org/${id}?size=200x200`}
        height="200"
        width="200"
        alt="robots"
      />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
