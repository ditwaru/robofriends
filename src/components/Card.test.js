import React from 'react';
import Card from './Card';
import { shallow } from 'enzyme';

it('expect to render card component', () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
