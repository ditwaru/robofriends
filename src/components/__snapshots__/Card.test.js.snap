// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5 tc"
>
  <img
    alt="robots"
    height="200"
    src="https://robohash.org/undefined?size=200x200"
    width="200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
